torch/autograd/gradcheck.py

```diff
@@ -305,7 +307,7 @@ def _get_numerical_jacobian(
 
 
 def get_numerical_jacobian(fn, inputs, target=None, eps=1e-3, grad_out=1.0):
-    """Deprecated API to compute the numerical Jacobian for a given fn and its inputs.
+    """Deprecate the API for computing numerical Jacobians of a given function and its inputs.
```

Comment:
This is not correct. "Deprecate" is not what the function does, the API is deprecated.

Reply:
Sorry. I have fixed this. Please review it.

@kit1980